docs/chat.md

```diff
@@ -190,12 +190,44 @@ See [OCP\RichObjectStrings\Definitions](https://github.com/nextcloud/server/blob
 
     - `talkMetaData` array:
 
-| field         | type   | Description                                                                                            |
-|---------------|--------|--------------------------------------------------------------------------------------------------------|
-| `messageType` | string | A message type to show the message in different styles. Currently known: `voice-message` and `comment` |
+| field         | type   | Description                                                                                                                                                          |
+|---------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `messageType` | string | A message type to show the message in different styles. Currently known: `voice-message` and `comment`                                                               |
+| `caption`     | string | A caption message that should be shown together with the shared file (only available with `media-caption` capability)                                                |
```

Comment:
Will the caption be markdown rendered?

Reply:
Last time I tested it with web frontend, markdown worked fine together, so don't see a reason to prevent it. Could it be tricky to realize on mobiles?

Reply:
No, would be fine, but if it should be just plain, I wouldn’t add markdown in the first place :)